test/datastore-test.js

```diff
@@ -0,0 +1,851 @@
+'use strict';
+
+/* global describe, it */
+
+var chai = require('chai');
+var expect = chai.expect;
+var chaiAsPromised = require('chai-as-promised');
+var dataStore = require('../src/models/datastore');
+var models = require('./data/models/Models');
```

Comment:
When I ran this on my linux box I had to change to lower case m for models/models.